templates/zerver/emails/missed_message.html

```diff
@@ -14,9 +14,10 @@
     Hello {{ name }},
 
     <p>
-        While you were away you received {{ message_count }} new message{{ message_count|pluralize }}{% if mention %} in which you were mentioned{% endif %}!
+        While you were away you received {{ message_count }} new{% if group_pm %} group{% elif private_message %} private{% endif %} message{{ message_count|pluralize }}{% if mention %} in which you were mentioned{% endif %}!
```

Comment:
This seems unrelated?

Reply:
As we remove message content from email body, there is no way to check type of message i.e. it's private messages, group messages or someone mentioned in stream.

Reply:
Just to check: can we ever have a missed message email with both PMs and @-mentions?

Reply:
If it's huddle message then it's first condition is true. If it's not then only it will check for second.

Reply:
I split this out as dfbe1e49149381ff0cedfd30daa788b457e5b411